RUFAS/routines/feed_storage/hay.py

```diff
@@ -5,6 +5,7 @@
 from RUFAS.weather import Weather
 
 from .storage import Storage
+from ...input_manager import InputManager
```

Comment:
```suggestion
from RUFAS.input_manager import InputManager
```
Relative imports should only be used to import modules from the same directory.